src/ServiceBus/ServiceBus/help/Test-AzServiceBusNameAvailability.md

```diff
@@ -0,0 +1,154 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.dll-Help.xml
+Module Name: Az.ServiceBus
+online version: https://docs.microsoft.com/en-us/powershell/module/az.servicebus/test-azservicebusnameavailability
+schema: 2.0.0
+---
+
+# Test-AzServiceBusNameAvailability
+
+## SYNOPSIS
+Checks the Availability of the given Queue or Topic name
+
+## SYNTAX
+
+### QueueCheckNameAvailabilitySet (Default)
+```
+Test-AzServiceBusNameAvailability [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String> [-Queue]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### TopicCheckNameAvailabilitySet
+```
+Test-AzServiceBusNameAvailability [-ResourceGroupName] <String> [-Namespace] <String> [-Name] <String> [-Topic]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Test-AzServiceBusNameAvailability** Cmdlet Check Availability of the provided Name of Queue or Topic
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Test-AzServiceBusNameAvailability -ResourceGroupName $resourceGroupName -Namespace $namespaceName -Name $nameQueue -Queue
+PS C:\> True
```

Comment:
@v-Ajnava nit: remove the `PS C:\>` from the beginning of this line

Reply:
Resolved